submissions/pashapushak1996/js-dom/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport"
+        content="width=device-width">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="reset.css" rel="stylesheet">
+  <link href="main.css" rel="stylesheet">
+  <script src="index.js"></script>
```

Comment:
Add this before the end of the tag `body`

Reply:
Thank you)